cloudbridge/cloud/providers/azure/azure_client.py

```diff
@@ -157,6 +158,7 @@ def __init__(self, config):
             tenant=config.get('azure_tenant')
         )
 
+        self._resource_client = None
```

Comment:
Redundant line?

Reply:
This is was an issue in my branch, it if fixed now.